test/scripts/openshift-ci/setup-e2e-tests.sh

```
@@ -54,255 +54,70 @@ if [[ "$RUNNING_LOCAL" == "true" ]]; then
 echo "SKLEARN_IMAGE=$SKLEARN_IMAGE"
-echo "KSERVE_CONTROLLER_IMAGE=$KSERVE_CONTROLLER_IMAGE"
-echo "LLMISVC_CONTROLLER_IMAGE=$LLMISVC_CONTROLLER_IMAGE"
-echo "KSERVE_AGENT_IMAGE=$KSERVE_AGENT_IMAGE"
-echo "KSERVE_ROUTER_IMAGE=$KSERVE_ROUTER_IMAGE"
 echo "STORAGE_INITIALIZER_IMAGE=$STORAGE_INITIALIZER_IMAGE"
 echo "ERROR_404_ISVC_IMAGE=$ERROR_404_ISVC_IMAGE"
 echo "SUCCESS_200_ISVC_IMAGE=$SUCCESS_200_ISVC_IMAGE"
 
-# Install Kustomize and yq
+# Install kustomize and yq (also needed for SeaweedFS kustomize build below)
 $PROJECT_ROOT/hack/setup/cli/install-kustomize.sh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)